pkg/action/scan.go

```diff
@@ -33,17 +33,18 @@ func findFilesRecursively(ctx context.Context, root string, c Config) ([]string,
 				return nil
 			}
 			// False positives in refs file
-			if strings.Contains(path, "/.git/") {
+			if strings.HasPrefix(path, ".git/") {
 				return nil
 			}
 			// Skip the bincapz directory if IgnoreSelf is true
```

Comment:
How about we remove this path check altogether - it's bound to keep shooting us in the foot; for example, when we rename the samples directory or add new ones. 

The intent was to ignore the bincapz binary, not anything relating to its source tree.

Reply:
Any objections to leaving that for a future improvement? 

This PR can just clean up the original `ignoreSelf` code and leave that for a future PR.